Sources/Testing/Parameterization/Test.Case.swift

```diff
@@ -156,10 +277,32 @@ extension Test {
 // MARK: - Codable
 
 extension Test.Parameter: Codable {}
-extension Test.Case.Argument.ID: Codable {}
+extension Test.Case.Argument.ID: Codable {
+  public init(from decoder: some Decoder) throws {
+    let container = try decoder.container(keyedBy: CodingKeys.self)
+
+    // The `isStable` property was added after this type was introduced.
+    // Previously, only stable argument IDs were ever encoded, so if we're
+    // attempting to decode one, we can safely assume it is stable.
+    let isStable = try container.decodeIfPresent(type(of: isStable), forKey: .isStable) ?? true
+
+    let bytes = try container.decode(type(of: bytes), forKey: .bytes)
+    self.init(bytes: bytes, isStable: isStable)
+  }
+}
 
 // MARK: - Equatable, Hashable
 
+extension Test.Case: Equatable, Hashable {
```

Comment:
This is publicly visible. Are you sure you want to commit to making instances of `Test.Case` equatable?

Reply:
Sure, that's not strictly needed here so I can remove it for now.